popSimManu.tex

```diff
@@ -364,37 +363,42 @@ \subsection*{The Species Catalog}
 %species divergence model of \cite{locke2011comparative}, which models isolation with migration between \emph{P. pygmaeus}
 %and \emph{P. abelii}. 
 %For \emph{E. coli} we have a simple, large constant population size model implemented.
-Beyond organism specific models \stdpopsim also includes a generic piecewise constant size model and
-isolation with migration (IM) model which can be used with any genome and genetic map combination.
+In addition to organism-specific models, \stdpopsim also includes a generic piecewise constant size model and
+isolation with migration (IM) model which can be used with any genome and genetic map.
 
-We note that we have put a good deal of effort into coming up with a standardized naming
+To guarantee reproducibility,
+we have standardized naming
 conventions for species, genetic maps, and demographic models that will enable long term stability
-of unique identifiers used throughout \stdpopsim. This is an essential feature to guarantee reproducibility.
+of unique identifiers used throughout \stdpopsim.
```

Comment:
should we give the readers a link to the documentation on this?

Reply:
a bit unwieldy, but ok: https://stdpopsim.readthedocs.io/en/latest/development.html#naming-conventions